symplyphysics/laws/quantum_mechanics/harmonic_oscillator/wave_eigenfunctions.py

```diff
@@ -0,0 +1,65 @@
+from sympy import Eq, Rational, sqrt, exp, factorial, pi
+from sympy.functions.special.polynomials import hermite
+from sympy.physics.units import hbar
+from symplyphysics import (
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+    units,
+    dimensionless,
+    angle_type,
+    symbols,
+    clone_symbol,
+)
+
+# Description
+## The time-independent Schrödinger equation for the harmonic potential can be solved using the
+## spectral method in the basis of Hermite functions.
```

Comment:
I see that this law is about harmonic oscillations but I do not see anything about oscillator in the description. Same about file name.